depthai_datatype_msgs/msg/IMUReportRotationVectorWAcc.msg

```diff
@@ -0,0 +1,10 @@
+# Not working because of the duplicate fields
```

Comment:
How does the C++ code work? C++ can also not have duplicated data member names

Reply:
I think C++ code is not working as expected.
I raised an issue https://github.com/luxonis/depthai-shared/issues/45

Reply:
Ack. Resolving this